docs/io.cozy.procedures.md

```diff
@@ -0,0 +1,31 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Procedures doctype
+
+## `io.cozy.procedures`
+
+The `io.cozy.procedures` doctype represents the administrative procedures.
+
+- `templateId`: {string} the id of the `io.cozy.procedures.templates` the procedure is related to.
+- `data`: {object} the data about the procedure (_e.g_ amount, duration, _etc_).
+- `personalData`: {object} the personal data associated to the procedure (_e.g_ email, phone number, _etc_).
```

Comment:
Je n'ai pas forcément mieux mais ça me semble étrange. Peut-être un "me", "contact" ou quelque chose du genre ?

Reply:
`me` ou `contact` on risque de croire que c'est une "copie" de io.cozy.contacts alors que c'est pas le cas. On va avoir des informations qui ne sont pas pertinents dans un `io.cozy.contacts`. 

`profil` peut-être?

Reply:
This two attributes `data` and `personalData` seem very generic. Why not put `email`, `phone number`, `amount`, `duration` as "real" attributes ?

Reply:
Ce doctype va servir à stocker les réponses à toutes les démarches administratives. Chaque démarche administrative demandera certainement des données différentes et ne seront donc pas figées. 

C'est le template qui définira ces informations. Du coup, ça me parait délicat d'avoir les "vrais" attributs directement à la racine. Non ?